VCS/1. User Guides/15. 管理資料磁碟.md

```diff
@@ -1 +1,212 @@
+:::
+
+## View the data disk
+
+View the ID, created time, name, size, type, state,  description, attached VCS instance, and mount path of your data disk.
+
+<!-- 1 start -->
+
+<details class="docspoiler">
+
+<summary><b>TWCC Portal</b></summary>
+
+<br>
+
+- Enter **Data Disk Management** page, then you can view the summary information of the diks. 
+
+![](https://cos.twcc.ai/SYS-MANUAL/uploads/upload_bd8f238dd56b9bd437f4b3c2dbbbccbf.png)
+
+
+- Click on one of your data disks and enter the **Data Disk Details** page, then you can view the detailed disk information.
+
+![](https://cos.twcc.ai/SYS-MANUAL/uploads/upload_a56cfee1ff28f578bff894daed2d5a62.png)
+
+
+
+</details>
+
+<!-- Space -->
+
+<div style="height:8px"></div>
+
+<!-- 2. start -->
+
+<details class="docspoiler">
+
+<summary><b>TWCC CLI</b></summary>
+
+<br>
+
+- View the ID, name, created time, mount path, size, state and type of the disk.
+
+```bash
+$ twccli ls vds
+```
+
+![](https://cos.twcc.ai/SYS-MANUAL/uploads/upload_c976443120fa105196269359143aeb3a.png)
+
+</details>
+
+## Attach to VCS instance
+
+After the data disk is created, you can attach the disk to the VCS instance then initialize the disk to use.
+
+<!-- 1 start -->
+
+<details class="docspoiler">
+
+<summary><b>TWCC Portal</b></summary>
+
+<br>
+
+*  After the data disk is created, it will appear at the top of the Data Disk Management list. When its state becomes **`AVAILABLE`**, you can attach to the instance for use. 
+
+![](https://cos.twcc.ai/SYS-MANUAL/uploads/upload_6da326c54d62392da23c299de1f3ed25.png)
+
+
+
+
+* Click the list to enter Data Disk Details page, and click **ATTACH**.
+
+![](https://cos.twcc.ai/SYS-MANUAL/uploads/upload_f025c6f70f0b5daa88ec8ea0dd3ee0dc.png)
+
+
+
+
+* When **Attach Disk** window pops up, select the VCS instance to be attached and click **OK**.
+
+:::info
+<i class="fa fa-paperclip fa-20" aria-hidden="true"></i> **Note:** The drop-down list only displays VCS instances that can be attached to data disks (with a state other than **`Stopped`**).
+:::
+
+
+![](https://cos.twcc.ai/SYS-MANUAL/uploads/upload_1ed0015b5ca27424909117bac12777e6.png)
+
+
+* After the data disk is mounted to the VCS instance, you can click **REFRESH**. The mounted instance will be displayed in the lower section, and the disk state will become **`IN-USE`**.
+
+![](https://cos.twcc.ai/SYS-MANUAL/uploads/upload_d79c9ac00e3961f1b1f23c7ed57883b5.png)
+
+
+</details>
+
+<!-- Space -->
+
+<div style="height:8px"></div>
+
+<!-- 2. start -->
+
+<details class="docspoiler">
+
+<summary><b>TWCC CLI</b> (Not yet supported)</summary>
+
+<br>
+
+</details>
+
+<br>
+
+
+:::info
+{%hackmd @docsharedstyle/note-en %}
+
+1. If you need to change to another disk type, or need to change the disk size, see [<ins>HowTo : Change data disk</ins>](https://man.twcc.ai/@twccdocs/howto-bss-replace-data-vol-en).
```

Comment:
[SUG]

If you need to change to another disk type, or need to change the disk size > 

For more information about changing disk types or disk sizes